### PR TITLE
Adjusted script to match correct structure for ro crate metadata json

### PR DESCRIPTION
**Pull request by MGS-sails**

### CODE REVIEW

**Review by SandyRogers — APPROVED** [review]
Thank you @MGS-sails ! Looks good to me - assuming the results crate JSON can be opened with the standard rocrate js and python libs 👍

File: scripts/prepare_motus_crates/motus_ro_crates_preparer.py [hunk 1]
```diff
@@ -5,13 +5,15 @@
...
 from urllib.parse import urljoin
+from uuid import uuid4
 
 import requests
 import shutil
 import tarfile
 from ro_crate_ui_assets_provider import RoCrateUIAssetsProvider
 from tqdm import tqdm
 from bs4 import BeautifulSoup
+from arcp import *
```
Comment (mberacochea) [inline scripts/prepare_motus_crates/motus_ro_crates_preparer.py]: A bit of a pedantic comment, I think it's better to do `from arcp import arcp_location` (which I think it's the only element from the arcp this code is using)

File: scripts/prepare_motus_crates/motus_ro_crates_preparer.py [hunk 2]
```diff
@@ -32,6 +34,7 @@ def __init__(self, original_ro_crate_path, destination_folder_path, extract_mult
         self.ro_crate_asset_provider = RoCrateUIAssetsProvider()
 
     def prepare_motus_ro_crate(self):
+        # self.setup_logging()
```
Comment (mberacochea) [inline scripts/prepare_motus_crates/motus_ro_crates_preparer.py]:
```suggestion
```

File: scripts/prepare_motus_crates/motus_ro_crates_preparer.py [hunk 3]
```diff
@@ -252,12 +320,12 @@ def extract_files_from_subdirectories(self, url):
 
 
 if __name__ == "__main__":
+    logging.info('feedback')
```
Comment (mberacochea) [inline scripts/prepare_motus_crates/motus_ro_crates_preparer.py]:
```suggestion
```

**Review by mberacochea — CHANGES_REQUESTED** [review]
Looks good to me, I just added a couple of very minor comments.
Also, `# "datePublished": datetime.datetime.now().strftime("%Y-%m-%d"),` can be removed I guess

File: scripts/prepare_motus_crates/motus_ro_crates_preparer.py (hunk 3 above)
Comment (MGS-sails) [reply scripts/prepare_motus_crates/motus_ro_crates_preparer.py]: Thanks Martin. 
I included these as part of my latest commit

**Comment by MGS-sails** [comment]
> Thank you @MGS-sails ! Looks good to me - assuming the results crate JSON can be opened with the standard rocrate js and python libs 👍

I've used ro-crate validator to confirm this. Initially there were a few errors. I've made adjustments to the script to correct those.

File: scripts/prepare_motus_crates/motus_ro_crates_preparer.py (hunk 1 above)
Comment (MGS-sails) [reply scripts/prepare_motus_crates/motus_ro_crates_preparer.py]: I've optimised this import as suggested.

Thanks

**Review by MGS-sails — COMMENTED** [review]
All suggestions have been implemented

**Pull request merged**